src/components/PostComments/PostComments.jsx

```diff
@@ -29,40 +29,36 @@ export class PostComments extends Component {
       <div className="post-comments">
         <div className="info">
           <span>
-            {commentsCount}
-            {' '}
-Comment
+            {`${commentsCount} `}
+            Comment
             {commentsCount !== 1 ? 's' : ''}
```

Comment:
So just an idea, instead of doing this am sure we will definitely need to pluralize some words other places, what if we create a helper that handles this? so it can be reusable

Reply:
@dusmel It's a good idea!